packages/run-protocol/src/vaultFactory/liquidation.js

```diff
@@ -63,17 +63,17 @@ const liquidate = async (
   ]);
   // NB: all the proceeds from AMM sale are on the vault seat instead of a staging seat
 
-  const [overage, shortfall] = AmountMath.isGTE(debt, proceeds.RUN)
+  const [overage, shortfall] = AmountMath.isGTE(debt, proceeds.Minted)
     ? [
         AmountMath.makeEmptyFromAmount(debt),
-        AmountMath.subtract(debt, proceeds.RUN),
+        AmountMath.subtract(debt, proceeds.Minted),
       ]
     : [
-        AmountMath.subtract(proceeds.RUN, debt),
+        AmountMath.subtract(proceeds.Minted, debt),
         AmountMath.makeEmptyFromAmount(debt),
       ];
 
-  const runToBurn = AmountMath.min(proceeds.RUN, debt);
+  const runToBurn = AmountMath.min(proceeds.Minted, debt);
   // debt is fully settled, with runToBurn and shortfall
```

Comment:
s/runToBurn/mintedToBurn/

Reply:
Reasonable suggestion, but also above and beyond the call of duty. `runToBurn` isn't visible via the contract API.